### DISCUSSION TITLE
Feature request:  Option to always include all facets (even when empty)

### DISCUSSION THREAD

**Discussion by trthingnes** [algolia/instantsearch#5573] [Ideas] [answered]
Wanted behavior: Display all product categories/vendors/manufacturers at all times (even when they contain no products).

Currently it works like this:
LG (10)
Samsung (2)

I want it to work like this:
AKAI (0)
LG (10)
Samsung (2)
Sony (0)

Can this be added as a configuration option on the index?

**Comment by Haroenv** [comment, marked as answer]
We don't have a way to return those facets automatically, but there's workarounds possible, by requesting the possible facet values a first time before searching: https://www.algolia.com/doc/guides/building-search-ui/widgets/customize-an-existing-widget/js/?language=js#display-facets-with-no-matches
